cloudberry/zion/src/main/scala/edu/uci/ics/cloudberry/zion/actor/RESTSolver.scala

```diff
@@ -26,7 +26,8 @@ class RESTSolver(val dataManager: ActorRef,
       val futureResult = Future.traverse(queries)(q => solveAQuery(q)).map(JsArray.apply)
       futureResult.map(result => (queries, result)).foreach { case (qs, r) =>
         out ! transform.transform(r)
-        qs.foreach(suggestViews)
+        //Disabled this suggest views to avoid competing resources in DB with ongoing ProgressiveSolver.
```

Comment:
I would suggest adding a TODO or FIXME so that it will be a little more noticeable.

Reply:
@JavierJia  Great! Thanks!